### PR TITLE
make the presubmit job shorter

### PR DESCRIPTION
**Pull request by richardwxn**
also fix the runner bug

### CODE REVIEW

File: perf/benchmark/configs/trialrun.yaml [hunk 1]
```diff
@@ -2,14 +2,14 @@ telemetry_mode: "dryrun"
...
   - p90
   - p99
-perf_record: true
+perf_record: false
 run_bothsidecar: true
 run_serversidecar: false
 run_clientsidecar: false
-run_baseline: true
\ No newline at end of file
+run_baseline: false
```
Comment (carolynhu) [inline perf/benchmark/configs/trialrun.yaml:15]: nit: add new line here

File: perf/benchmark/run_benchmark_job.sh [hunk 2]
```diff
@@ -254,7 +254,6 @@ for dir in "${CONFIG_DIR}"/*; do
     # TRIALRUN as a pre-submit check, only run agaist the first set of enabled perf run in the perf_conf file
     if [[ "${TRIALRUN}" == "True" ]]; then
        run_benchmark_test "${WD}/configs/trialrun.yaml"
-       collect_flame_graph
```
Comment (carolynhu) [inline perf/benchmark/run_benchmark_job.sh:257]: why delete this? we could also have this tested in the presubmit job I do think

Comment (carolynhu) [reply perf/benchmark/run_benchmark_job.sh:257]: so to make the above perf_record enabled?

Comment (richardwxn) [reply perf/benchmark/run_benchmark_job.sh:257]: Let's focus on the main part(installation, setup, runner) first

File: perf/benchmark/configs/trialrun.yaml (hunk 1 above)
Comment (richardwxn) [reply perf/benchmark/configs/trialrun.yaml]: done

File: perf/benchmark/run_benchmark_job.sh (hunk 2 above)
Comment (carolynhu) [reply perf/benchmark/run_benchmark_job.sh:257]: ok, other cases can be added in the following PRs.

**Pull request merged**